### ISSUE TITLE
which server would you advise?

### ISSUE THREAD

**Issue by jrmaciel** [MetacoSA/QBitNinja#86]
my server is very slow, do you advise with more memory or processing? the qbit console is overloading, I currently have an A2m_v2

**Comment by NicolasDorier** [comment]
You mean for the indexer part?
I don't know, check the task manager to see what is the bottleneck.

**Comment by jrmaciel** [comment]
my server have 16 of memory and 2 processors

**Comment by NicolasDorier** [comment]
Are you indexing step or it is like this after indexed everything?

**Comment by jrmaciel** [comment]
yes it is working, the only problem is the lack of memory and then it causes the processes to be slow

which server do you use in api.qbit.ninja?

**Comment by NicolasDorier** [comment]
I used `Standard D1 v2 (1 vcpus, 3.5 GB memory)`

![image](https://user-images.githubusercontent.com/3020646/51328753-69ed2f80-1ab7-11e9-9b24-a2423042cef5.png)

**Comment by NicolasDorier** [comment]
I noticed that putting bitcoin core on the OS drive speed up things a lot.

**Comment by NicolasDorier** [comment]
You need to run "even if not logged in" on Task scheduler so the console does not show up.

**Comment by jrmaciel** [comment]
but still the memory remains very high
this is how it is not indexing right

**Comment by NicolasDorier** [comment]
As I said, the CPU problem is probably just transaction ids printing.
Use the task scheduler so  QBitNinja indexer run without printing output to a console.

**Comment by jrmaciel** [comment]
after running in output to a console my balance has a problem
look 

https://btccmd.azurewebsites.net/balances/1P8tS7wZ2qn9JkCf48GVkkqovXR3y3kmTm/summary

http://api.qbit.ninja/balances/1P8tS7wZ2qn9JkCf48GVkkqovXR3y3kmTm/summary

**Comment by jrmaciel** [comment]
the problem was in my web application, the only problem now that shows the new transactions but the balance is problem

**Comment by jrmaciel** [comment]
restart qbit pls, I'm using it while I work on my

**Comment by NicolasDorier** [comment]
Why restart? it is working.

For the difference of values, did the initial indexing really finished? It takes several days.

**Comment by jrmaciel** [comment]
why I try to get a balance or something and it's not updated and also when I try to send a transaction it takes about 25 minutes to send

this is only while I finish indexing my server again

**Comment by jrmaciel** [comment]
yes it was over, it was working but I think it gave some problems in the blocks and it did not update the balances correctly.
he just pulled 135 queue, I think the blocks that gave problem
